Makefile

```diff
@@ -16,6 +17,9 @@ include tools/git-verify-nodiff/rules.mk
 include tools/golangci-lint/rules.mk
 include tools/goreview/rules.mk
 include tools/semantic-release/rules.mk
+include tools/eslint/rules.mk
+
+example_gen_path := examples/proto/gen/typescript/einride/example/freight/v1
```

Comment:
since this is only used in one place I would actually inline it to make things faster to read without jumping around

Reply:
Yup I agree! 🚀